src/components/sides/TaskInfo.vue

```diff
@@ -267,6 +273,9 @@ import Spinner from '@/components/widgets/Spinner'
 import SubscribeButton from '@/components/widgets/SubscribeButton'
 import TaskTypeName from '@/components/widgets/TaskTypeName'
 import PreviewPlayer from '@/components/previews/PreviewPlayer'
+import csv from '../../lib/csv'
```

Comment:
Can you use '@' import? And place the lib import before the widget imports?

Reply:
I have changed all relatives imports with '@' imports